agent/consul/leader_connect_test.go

```diff
@@ -314,18 +314,6 @@ func TestCAManager_Initialize_Secondary(t *testing.T) {
 	}
 }
 
-func waitForActiveCARoot(t *testing.T, srv *Server, expect *structs.CARoot) {
```

Comment:
This was basically a dupe of `testrpc.WaitForActiveCARoot`, so I removed this one.